stac_validator/stac_validator.py

```diff
@@ -10,7 +10,11 @@
 
 @click.command()
 @click.argument("stac_file")
-@click.option("--lint", is_flag=True, help="Use stac-check to lint stac object.")
+@click.option(
+    "--lint",
+    is_flag=True,
+    help="Use stac-check to lint stac object instead of validating it.",
```

Comment:
@gadomski stac-check does do validation as well. It uses stac-validator for basic validation. It could be set up similar to how it is set up in stactools, using only the linting logic but right now in stac-validator it does display validation information.

Reply:
Ah, gotcha. Yeah I was just reading the code path, where it looked like it was an either/or situation. IMO it'd be clearer if you were able to keep the responsibilities separate; validator calling linter calling validator is a bit funky.

If we want to keep things are they are now, I'll make the help message more explicit, e.g. `"Use stac-check to lint the stac object in addition to validating it."`

Reply:
Hi. If you change the message to note that linting also calls validation I think that would be good. You're right about keeping responsibilities separate. I think I should use stac lint in stac-validator like it's used in stactools and only call the linting logic.